src/pe_source/intelx_identity.py

```diff
@@ -0,0 +1,289 @@
+"""Collect Intelx credential leak data."""
```

Comment:
Use consistent capitalization:
```suggestion
"""Collect IntelX credential leak data."""
```